docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -0,0 +1,257 @@
+---
+approvers:
+- davidopp
+- kevin-wangzefeng
+- bsalamat
+title: Taints and Tolerations
+---
+
+Node affinity, described [here](/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature),
+is a property of *pods* that *attracts* them to a set of nodes (either as a
+preference or a hard requirement). Taints are the opposite -- they allow a
+*node* to *repel* a set of pods.
```

Comment:
To reviewers: line 9-12 are new. They provide an anchor back to the node-affinity feature for comparison.